src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -1597,4 +1597,89 @@ String getApiTailUrl(String tail) {
         if (tail.length()>0 && !tail.startsWith("/"))    tail='/'+tail;
         return "/repos/" + getOwnerName() + "/" + name +tail;
     }
+
+    public static class IssueEventInfo {
```

Comment:
@martinvanzijl 
Does this need to be a child class inside `GHRepository`?

Reply:
@bitwiseman 
I've moved it to a separate class called `GHIssueEvent`.